### ISSUE TITLE
How can users share comments and saved filters with their team?

### ISSUE THREAD

**Issue by jaclynperrone** [hammerlab/cycledash#673]
Let's talk about the different options we have. I heard a lot of feedback from researchers about how they need a way to communicate their findings/variant flags/comments with the rest of their team. And sometimes with just one person in particular.

It sounds like having account creation would solve this (either individual- or team-based logins), but not sure it's the best way to go? It sounds like it would add a ton of complexity but I could be wrong. All of this also depends on who we define as our users [#654]

**Comment by ihodes** [comment]
I wouldn't worry too much about the complexity of a login system; a rudimentary one with a basic ACL wouldn't be difficult to get in place. As we get more complicated, it could get more difficult.

I thought @tavinathanson had an issue open for this, but I couldn't find it.

**Comment by jaclynperrone** [comment]
@ihodes that's exactly the answer I was hoping for!

**Comment by tavinathanson** [comment]
I thought I did too, but also couldn't find it.

My 2c: a login system seems fine/good as long as we have a specific use case in mind (and it sounds like we're moving in that direction). I had wanted to avoid throwing it before it was necessary.

**Comment by jaclynperrone** [comment]
Yes! Completely agree. It's good to hear that it's not off the table.
